indexer/resolver.go

```diff
@@ -0,0 +1,15 @@
+package indexer
+
+import (
+	"context"
+
+	"github.com/quay/claircore"
+)
+
+// Resolver is used for any reasoning that needs to be done with all the layers in context.
+//
+// Resolvers are called at the end of the coalesce step when reports
```

Comment:
Is this the indexing equivalent to Enricher?

Reply:
Kind of, but with the distinction that a `Resolver` is (potentially) modifying the result once the layer indexing has finished. Enrichers really only add data.